ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -35,7 +35,7 @@ export const AvatarAccount = ({
         <BlockieIdenticon
           className={classnames('mm-avatar-account__blockie')}
           address={address}
-          diameter={DIAMETERS[size]}
+          diameter={Number(DIAMETERS[size])}
```

Comment:
Fix for <img width="1508" alt="Screenshot 2022-12-14 at 4 08 05 PM" src="https://user-images.githubusercontent.com/39872794/207573104-bf4beeda-dbe4-497b-97db-4dd20e8f6e3f.png">